src/applications/mhv-supply-reordering/containers/ConfirmationPage.jsx

```diff
@@ -32,42 +56,84 @@ export const ConfirmationPage = () => {
         />
       </div>
 
-      <va-alert status="success" ref={alertRef}>
-        <h2 slot="headline">Your application has been submitted</h2>
-      </va-alert>
-
-      <p>We may contact you for more information or documents.</p>
-      <p className="screen-only">Please print this page for your records.</p>
-      <div className="inset">
-        <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
-          Order Medical Supplies Claim{' '}
-          <span className="vads-u-font-weight--normal">(Form {formId})</span>
-        </h3>
-        {fullName ? (
-          <span>
-            for {fullName.first} {fullName.middle} {fullName.last}
-            {fullName.suffix ? `, ${fullName.suffix}` : null}
-          </span>
-        ) : null}
-
-        {confirmationNumber ? (
-          <>
-            <h4>Confirmation number</h4>
-            <p>{confirmationNumber}</p>
-          </>
-        ) : null}
-
-        {isValid(submitDate) ? (
-          <p>
-            <strong>Date submitted</strong>
-            <br />
-            <span>{format(submitDate, 'MMMM d, yyyy')}</span>
-          </p>
-        ) : null}
-
-        <va-button onClick={window.print} text="Print this for your records" />
+      <VaAlert status="success" class="vads-u-margin-bottom--4" ref={alertRef}>
+        <h2 slot="headline">You’ve submitted your medical supplies order</h2>
+        <p>
+          We’ll send you a confirmation email about your order to {emailAddress}
+          . Your order will be processed in the next business day.
+        </p>
+      </VaAlert>
+
+      <VaSummaryBox>
+        <h3 slot="headline">Your order information</h3>
+
+        <h4>Who submitted this form</h4>
+        <p className="vads-u-margin-y--0 dd-privacy-mask">
+          {fullName ? (
+            <span>
+              {fullName.first} {fullName.middle} {fullName.last}
+              {fullName.suffix ? `, ${fullName.suffix}` : null}
+            </span>
+          ) : null}
+        </p>
+
+        <h4 className="vads-u-margin-top--1">Confirmation number</h4>
+        <p className="vads-u-margin-y--0">{orderIds}</p>
+
+        <h4 className="vads-u-margin-top--1">Supplies ordered</h4>
+        <ul className="vads-u-margin-top--0">
+          {productNames.map(name => (
+            // eslint-disable-next-line react/jsx-key
+            <li>{name}</li>
```

Comment:
Adding a key attribute to remove the warning `ConfirmationPage.jsx:85 Warning: Each child in a list should have a unique "key" prop.`